Resources/doc/reference/installation.rst

```diff
@@ -120,6 +120,23 @@ dashboard. To be able to use it, make sure it's enabled on SonataBlockBundle's c
     what a block is. SonataBlockBundle is a useful tool, but it's not vital
     that you understand it right now.
 
+
+Language localization
+---------------------
+
+In order to have localized menu and dashboard labels, set up the translation strategy:
+
+.. code-block:: yaml
+
+    # app/config/config.yml
+    framework:
+        (...)
```

Comment:
For better yaml readability, please remove it or prefix it with a `#` comment like `# (...)`.